### DISCUSSION TITLE
Update Slider value in iOS SDK

### DISCUSSION THREAD

**Discussion by tiwari1amrit** [ant-media/Ant-Media-Server#5154] [Q&A]
I have to add a slider view in my iOS app, I can't find anything that updates my slider value. And how can I forward the video or backward using the slider? Did anyone implement it?
Like this image. 

<img width="609" alt="Screenshot 2023-05-27 at 10 29 36 am" src="https://github.com/ant-media/Ant-Media-Server/assets/22368948/038efb7c-7acf-4652-9c7b-5f753c821927">

Environment
iOS app development via Swift programming language

**Comment by tremzo** [comment]
@tiwariammit Is this on a VoD asset or are you referring to the live stream? This looks more like a player functionality. Which player are you using?

**Reply by tiwari1amrit** [reply]
Hi @timantmedia 
Currently, I am using Ant media player. I want to add feature of let user able to scroll slider to see back in a live stream. And later on, I am going to play recorded video as well. I didn't find anything regarding changing the video rendering like fast forward or backward. How can I achieve in this Ant Media.
